src/botPage/view/blockly/index.js

```diff
@@ -317,6 +317,31 @@ export default class _Blockly {
     load(blockStr = '', dropEvent = {}) {
         let xml;
 
+        const validateXML = xml => {
+            if (document.implementation.createDocument) {
```

Comment:
- Is there a reason for checking whether we can create an `XMLDocument`? We can remove this condition as `DOMParser.parseFromString` does the checking.
- We don't use the `validateXML` function elsewhere so, in my opinion, we can just run through the logic right away.